ofctrl/ofctrl.go

```diff
@@ -94,6 +94,10 @@ type AppInterface interface {
 
 	// Controller received a multi-part reply from the switch
 	MultipartReply(sw *OFSwitch, rep *openflow15.MultipartReply)
+
+	FlowGraphEnabledOnSwitch() bool
+
+	TLVMapEnabledOnSwitch() bool
```

Comment:
Why adding these two methods?

Reply:
It is to provide more options to the AppInterface to decide if the tlvMap and the flowGraph ( tableDb/groupDb/meterDb ) is used or not.